wiki/Installation-guide.md

```diff
@@ -1,114 +1,84 @@
 # Installation guide
 
-## Basic installation
+## Launcher (BETA)
 
-PC side:
+Launcher will allow you to manage old, current and new installations of ALVR streamer and allow to automatically install and upgrade to specific ALVR client version on headset
 
-* Install SteamVR, **launch it once** then close it. This is to make sure it sets the environment correctly for ALVR.
-* Go to the latest release [download page](https://github.com/alvr-org/ALVR/releases/latest). In the "Assets" section at the bottom download the ALVR Installer.
-* Run the installer. If prompted, allow the execution in the SmartScreen popup. You need to give administrator permissions to install ALVR. For best compatibility do not change the installation folder.
-* Once the installation finished, launch ALVR. You are greeted with a setup wizard. Follow the setup to set the firewall rules and presets.
+### Installation
 
-**If you have problems launching ALVR, follow the guide below to use the portable version**
-
-Headset side:
-
-* Install SideQuest on your PC and enable developer mode on the headset. You can follow [this guide](https://sidequestvr.com/setup-howto).
-* Connect your headset to Sidequest. If you have an Oculus Quest 1/2 download the ALVR app [here](https://sidequestvr.com/app/9), if you have an Oculus Go download it [here](https://sidequestvr.com/app/2658)
+* Download `alvr_launcher_windows.zip` on windows or `alvr_launcher_linux.tar.gz` from the release [download page](https://github.com/alvr-org/ALVR/releases/latest) and extract into a path that contains only ASCII characters (english only) and has edit permissions without administrator or root rights.
+* Run `ALVR Launcher.exe` (on Windows) or `alvr_launcher_linux/ALVR Launcher` (on Linux)
+* Press `Add version` button
+* For default installation keep channel and version as is and press `Install`
+* Wait until it finishes downloading, installing (depends on your connection)
+* To install ALVR client on headset, use button `Install APK`
+* In the list, to open streamer app (PC) press `Launch`. You will be greeted with a setup wizard. Follow the setup to set the firewall rules and other settings.
 
 ### Usage
 
-* Launch ALVR on your headset. While the headset screen is on, click `Trust` next to the client entry (on the PC) to start streaming.
+* Before launching SteamVR through ALVR, please install it and run at least once without ALVR and close it.
```

Comment:
As per comment above, this line *should* be changed, but otherwise will not hurt.